semver/comparator_format.ts

```diff
@@ -7,6 +7,8 @@ import { format } from "./format.ts";
  * @example >=0.0.0
  * @param comparator
  * @returns A string representation of the comparator
+ *
+ * @deprecated (will be removed in 0.213.0) Use {@linkcode formatRange} instead.
```

Comment:
```suggestion
 * @deprecated (will be removed in 0.213.0) Use {@linkcode SemVerRange} instead of {@linkcode Comparator} and {@linkcode formatRange} for formatting it.
```